sklearn/datasets/openml.py

```diff
@@ -489,24 +589,41 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         If True, returns ``(data, target)`` instead of a Bunch object. See
         below for more information about the `data` and `target` objects.
 
+    return_frame : boolean, default=False
+        If True, returns a Bunch where the data attribute is a pandas
+        DataFrame.
+
+    chunksize : int, default=5000
+        Number of rows to read at a time when constructing a dataframe.
+        Only used when ``return_frame`` is True.
```

Comment:
we could be using this more generally instead of the messy np.fromiter code.

Reply:
What do you find messy about the `np.fromiter` code?